README.adoc

```diff
@@ -29,58 +35,62 @@ curl https://sh.rustup.rs -sSf | sh
 sudo apt install make clang pkg-config libssl-dev
 ----
 
-Then, install Polkadot PoC-2:
+=== Install PoC-3 "Alexander" Testnet
+
+Install Polkadot PoC-3 and have a `polkadot` binary installed to your `PATH` with:
 
 [source, shell]
-cargo install --git https://github.com/paritytech/substrate.git --branch v0.2 polkadot
+cargo install --git https://github.com/paritytech/polkdot.git --branch v0.3 polkadot
```

Comment:
I think `pokdot` is a typo here.
Maybe we should fix that since most people will probably copy&paste this code.